packages/core/.eslintrc

```diff
@@ -0,0 +1,40 @@
+{
+  "env": {
+    "browser": true,
+    "node": true,
+    "jest": true
+  },
+  "extends": [
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended"
+  ],
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "legacyDecorators": true
+    },
+    "project": "./tsconfig.json"
+  },
+  "plugins": ["@typescript-eslint", "prettier", "jest"],
+  "rules": {
+    "prettier/prettier": ["error", {
+      "trailingComma": "all",
+      "singleQuote": true,
+      "printWidth": 100,
+      "arrowParens": "avoid"
```

Comment:
why? Since `Prettier v2.0` they changed the default to `always`, IMO the explanation is legit:

> At first glance, avoiding parentheses may look like a better choice because of less visual noise. However, when Prettier removes parentheses, it becomes harder to add type annotations, extra arguments or default values as well as making other changes. Consistent use of parentheses provides a better developer experience when editing real codebases, which justifies the default value for the option.

Reply:
This has been copied from our application's eslint config (as other config variables). But because this is a greenfield project we can do with default settings.